app/src/main/java/com/example/unsplashphotos/data/repository/DownloaderUtils.kt

```diff
@@ -0,0 +1,73 @@
+package com.example.unsplashphotos.data.repository
+
+import android.annotation.SuppressLint
+import android.app.DownloadManager
+import android.content.Context
+import android.database.Cursor
+import android.net.Uri
+import android.os.Environment
+import android.widget.Toast
+import com.example.unsplashphotos.R
+import java.io.File
+
+class DownloaderUtils(private val context: Context) {
```

Comment:
Try to use object declaration mentioned in ShareUtils. Pass context as method parameter instead of constructor parameter.